README.md

```diff
@@ -40,4 +40,4 @@ for client libraries to work.
 ## How to run the application tests
 
 1. Set up the emulator as described in #1 above.
-2. Run `mvn test`.
+2. Run `mvn integration-test`.
```

Comment:
I see some logic now in the test which uses a system property for JDBC. Does executing mvn integration-test run both jdbc and non-jdbc versions of the test ? Please update the README as appropriate.

Reply:
They need to be run separately at the moment. The README now contains instructions for how to do so.